components/config/color_picker.py

```diff
@@ -30,40 +30,40 @@ def get_component(columns):
     ])
 
 
-def get_colors(data, color_column):
+def get_colors(data, color_params):
     """Return an array of colors, one for each row of data, and
     a dictionary for building a colorbar.
-    If the color column argument is None, default color 'red' is
-    used instead.
+    If the color parameters argument is None, default color 'red'
+    is used instead.
 
     The values in the given column are normalized to the range
-    [0,1], and the result is used to extract a color from a
-    colormap.
+    [0,1] using the given minimum and maximum values. The result
+    is used to extract a color from a colormap.
 
     Keyword arguments:
     data -- Dataframe object
-    color_column -- Column to use for calculating the colors,
-        or None for default values
+    color_column -- A tuple with the information for extracting
```

Comment:
This should be changed to color_params. Also, should we document what this tuple contains? (i.e. name, minimum, and maximum)